examples/kafka/kafka-persistent-single.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 1
+    listeners:
+      plain: {}
+      tls: {}
+    config:
+      offsets.topic.replication.factor: 3
```

Comment:
All these options should be set to 1. With single Kafka broker, you cannot have anything with replication factor and ISR bigger then 1.